swagger.yaml

```diff
@@ -2390,6 +2432,10 @@ paths:
           description: Error.
           schema:
             $ref: '#/definitions/error'
+        503:
+          description: Unavailable.
+          schema:
+            $ref: '#/definitions/error'
```

Comment:
Why add these other errors?

Reply:
Mmmm I thought we needed 503 here too, but we don't - will remove